### ISSUE TITLE
[URGENT MY CUSTOMERS WAITING FOR UPDATE I CAN´T MAKE] 😬 Unexpected error while publishing to web

### ISSUE THREAD

**Issue by Toninomex** [FlutterFlow/flutterflow-issues#3642]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

<img width="1511" alt="Captura de pantalla 2024-08-16 a la(s) 9 00 52 p m" src="https://github.com/user-attachments/assets/fc367977-a404-4614-a97b-437ff5da82b4">
 
I was publishing as always to the web a suddenly I get this error. After pushing to see what was all about,  was driven to the App Assets page where I can do nothing.

### Expected Behavior

To be able to publish to the web as usual.

### Steps to Reproduce

I just was upadating my app to the web because I add a new page called "pendientes terceros" and suddenly this happen.

I even went back to a previous snapshot before creating that page and the error still there (and not shown as an error during the snapshot)

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4kleflsJNgocNG15P5bflKpCc7Q1AnTLIFse0aRR4gF5DtB6QMesmkX0NKT9/jdH5mDleimj4Ep/zljdnpFe4oZE+BQaJD0JVATjvxTjmhaYSkDKuaVnBQM8pnBW3H37uFphJACM9fSloA2kOpN63qNleeY8aSfxBlZ7vfcPo=

### Visual documentation

<img width="1511" alt="Captura de pantalla 2024-08-16 a la(s) 9 00 52 p m" src="https://github.com/user-attachments/assets/133268b5-2e29-441c-8e05-15e2994c0347">

### Environment

```markdown
- FlutterFlow version: 4.1.81
- Platform: web
- Browser name and version: chrome
- Operating system and version affected: Mac Sonoma 14.5
```

**Comment by paulperez-dev** [comment]
Hi @Toninomex! Can you please verify if the font mentioned in the error is uploaded (see attached image)?

<img src="https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/99ad4ec8-e2eb-4534-bc65-b1215c5c1028/37a8efda-8c59-4e33-b18a-84edb3a70efd?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS85OWFkNGVjOC1lMmViLTQ1MzQtYmM2NS1iMTIxNWM1YzEwMjgvMzdhOGVmZGEtOGM1OS00ZTMzLWIxOGEtODRlZGIzYTcwZWZkIiwiaWF0IjoxNzI0MDE3ODYwLCJleHAiOjMzMjk0NTc3ODYwfQ.U1isRhTsV6feV5YphCRRuR4xmvGDjfU4X9tcb_bJjbI " alt="image.png" width="1680" height="817" />

**Comment by Toninomex** [comment]
Hi Paul
This is what I have… 
I am not using and I never used “MyFlutterApp.ttf

Missing custom font asset:
"MyFlutterApp.ttf"
Please re-upload or delete font entry.
￼

**Comment by Toninomex** [comment]
I sent you vía mail the screen shot, did you receive it?
I am using Poppins I did never used "MyFlutterApp.ttf"
I did not make any different than what I have been making so I believe is a bugg not related with any font at all.
I read on the forums that other people have moreless the same problem, but they didn´t post the solution.
I need to solve this ASAP as clients are waiting for an update and the system, doesn´t allow me until I solve that error
what can we do?

<img width="1183" alt="image" src="https://github.com/user-attachments/assets/d233e92a-c38c-41b0-96c7-398f273a6fd5">
